packages/mdc-base/component.ts

```diff
@@ -1,6 +1,6 @@
 /**
  * @license
- * Copyright 2016 Google Inc.
+ * Copyright 2019 Google Inc.
```

Comment:
Leave copyright years as-is since we're not changing the actual innards of these modules.